MojioSDK/mojiosdksrc/src/main/res/values-mcc230-mnc01/configs.xml

```diff
@@ -0,0 +1,4 @@
+<resources>
+    <string name="endpoint_url">cz-api.moj.io</string>
+    <string name="sandbox_available">false</string>
```

Comment:
I think you could define this as <bool name="..."> and load with getResources().getBoolean()